app/screens/clock_display/index.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import {getTheme, get as getPreference} from 'mattermost-redux/selectors/entities/preferences';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import Preferences from 'mattermost-redux/constants/preferences';
+
+import ClockDisplay from './clock_display';
+
+function mapStateToProps(state) {
+    const militaryTime = getPreference(state, Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time') || 'false';
```

Comment:
I think it would be better if we use the `getBool` from the preference selectors here instead https://github.com/mattermost/mattermost-redux/blob/master/src/selectors/entities/preferences.js#L29

Reply:
Actually, after looking at this again. The reason I chose getPreference was that it returns the boolean as a string.  Which is the format the the Components expect.

It's better to reason when you are checking for it in this format in my opinion:
```
checked: {militaryTime == 'false'}
or 
checked: {militaryTime == 'true'}
```

Reply:
If it was a boolean, wouldn't you just have `checked: {militaryTime}` and `checked: {!militaryTime}` then?